schemas/date.schema.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+
+  "type":"string",
+  "pattern":"^\d{4}-((0[1-9])|(1[012]))-((0[1-9]|[12]\d)|3[01])$"
```

Comment:
@hajoeichler because 2015-00-32 would be valid for your regex, but not in the other one.
Actually I'm also fine with `d{4}-d{2}-d{2}` - it's much quicker to understand and all sorts of invalid dates will not be able to be catched with a regex anyway (leap years and what not :wink:)